ISOLATION.md

```diff
@@ -45,5 +45,5 @@ The most interesting pieces of the multitenant plugin are:
 
 * **ovssubnet/controller/multitenant/bin/openshift-sdn-multitenant-setup.sh**: this script is run every time the openshift-node process starts or stops.  It does initial setup, like configuring the lbr0 bridge, adding the OVS bridge, adding the OVS VXLAN port, setting up the tun0 port and NAT rules, and configuring the non-pod-specific OVS rules.
 
-* **ovssubnet/controller/multitenant/multitenant.go**: this module watches etcd for indications of minions added to or removed from the cluster, and updates the OVS rules to ensure each minion can be reached through the VXLAN tunnel.
+* **ovssubnet/controller/multitenant/multitenant.go**: this module watches etcd for indications of nodes added to or removed from the cluster, and updates the OVS rules to ensure each node can be reached through the VXLAN tunnel.
```

Comment:
Functional/Non-functional changes are already segregated, node-> minion change is a separate commit in this pr